### DISCUSSION TITLE
2024-07-17 수요일

### DISCUSSION THREAD

**Discussion by Captain-Kim** [nbc-final-a10/TripBuddy#48] [기록보드]
2024-07-17 수요일 기록보드입니다.

## 여행가고십조(A10) 일일 공통 일과표

| 시간 | 일과 |  기록 |
| :-: | :-: | :-: |
| 09:00 ~ 10:00 | 기술면접 / 답안작성 | |
| 10:00 ~ 10:30 | 하루 일과 준비 | |
| 10:30 ~ 13:00 | User Flow 작성 및 역할 분배 | <img src="https://github.com/user-attachments/assets/46ab008b-cbbd-43cb-acd3-eb68ef8695ec" weight="300" height="300"/> |
| 13:00 ~ 14:00 | 점심 식사 | |
| 14:00 ~ 15:00 | 팀 공통, 개인별 챌린지 과제 토의 | <img src="https://github.com/user-attachments/assets/3fe2e397-b474-4573-a966-13cc183db117" weight="300" height="300"/> |
| 15:00 ~ 16:00 | User Flow 작성 | <img src="https://github.com/user-attachments/assets/7baaf59f-800f-4ed9-a025-59b781605003" weight="300" height="300"/> |
| 16:00 ~ 17:00 | 컴포넌트 구조 작성, 제작 담당자 분배 | <img src="https://github.com/user-attachments/assets/8e6b2dc2-97c8-47c1-ac16-10572a236cab" weight="300" height="300"/> |
| 17:00 ~ 18:00 | DB 테이블 작성 | <img src="https://github.com/user-attachments/assets/eed69686-d12b-45f1-b184-b359246ff9d5" weight="300" height="300"/> |
| 18:00 ~ 19:00 | 저녁 식사 | |
| 19:00 ~ 20:00 | API 명세서 작성 | <img src="https://github.com/user-attachments/assets/1e00c42e-178d-42ed-88cc-e4df56096033" weight="300" height="300"/> |
| 20:00 ~ 21:00 | 5분 기록보드 작성 / 개인 회고 | <img src="https://github.com/user-attachments/assets/0c565c42-4436-4b7c-9eb9-1b884c6fc767" weight="300" height="300"/> |

### 오늘 한 일

- 본인인증 외부 API 응답값 확인 및 테스트
```
// 사진
<img src="이미지 주소" weight="300" height="300"/>
```

### 내일 계획

- supabase auth 비밀번호 찾기 기능 연결

**Comment by eunohhh** [comment]
## 개인 기록보드 서식 (오은)

### 오늘 한 일

- 유저 플로우 작성
- webRTC 테스트
- 페이지별 업무 분배
- 공통 및 개인별 챌린지 과제 논의
- DB 테이블 작성
- API 명세서 작성

### 내일 계획
- Supabase DB 세팅
- 디자이너 미팅
- webRTC 연구

**Comment by Captain-Kim** [comment]
## 개인 기록보드 서식 (김병준)

### 오늘 한 일

- User Flow 작성
- 컴포넌트 구조 작성
- 페이지별 업무 분배
- 공통 및 개인별 챌린지 과제 논의
- DB 테이블 작성
- API 명세서 작성
- Supabase 팀원 초대

### 내일 계획

- Supabase DB 세팅
- 디자이너 미팅
- Rich-Text-Editor 레퍼런스 탐색

**Comment by slsl2** [comment]
### 오늘 한 일 (같이)
- 유저 플로우 작성
- 구현 기능 정리
- 컴포넌트 구조 작성 및 업무 분배
- DB 테이블 작성
- API 명세서 작성

### 오늘-내일 할 일 (개인)
- 채팅 구현 강의 보기
- 기술면접 연습

**Comment by Kyara0324** [comment]
### 오늘 한 일 (같이)
- User Flow 작성
- 페이지별 업무 분배
- 공통 및 개인별 챌린지 과제 논의
- DB 테이블 작성
- API 명세서 작성

### 내일 할 일
- Supabase DB 세팅
- 디자이너 미팅
- 기술면접 연습 및 기술면접 보기

**Comment by hyunjeongkwak** [comment]
<h3>오늘 한 일</h3>

- user flow 제작
- 챌린지 과제 구상 및 분배 후 컴펌
- 과제에 따라 페이지 분배
- DB 테이블 작성
- API 명세서 작성

<h3>내일 할 일</h3>

- 기술 면접 준비
- 디자이너님과 팀 회의 진행
- 기술 면접
